todos-app/src/App/ListFooter/ListFooter.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { useState } from "react";
+import { ITodo, TodoListType } from "../../DataStructure";
+import FilterList from "./FilterList/FilterList";
+
+const ListFooter: React.FC<TodoListType> = ({ todos }) => {
+  // const [todos, setTodos] = useState<ITodo[]>([]);
+  const completed: number = todos.filter(
```

Comment:
nit, `completedCount`

Reply:
Fixed it. Rename variable _completed_ to _completedCount_. And It's the same with the _backlog_ variable